controllers/vaultresourcecontroller/utils.go

```diff
@@ -21,7 +21,7 @@ import (
 	"time"
 
 	"github.com/redhat-cop/operator-utils/pkg/util"
-	"github.com/redhat-cop/operator-utils/pkg/util/apis"
+	"github.com/redhat-cop/operator-utils/pkg/util/apis" // TODO - are we trying to remove the apis dependency and its status completely since we are using the ReconcileSuccessful constant instead?
```

Comment:
@raffaelespazzoli are we trying to remove the apis dependency and its status constants completely since we are using the ReconcileSuccessful constant instead? seems inconsistent

Reply:
Goal is to remove this dependency